blossom/templates/website/generic_form.html

```diff
@@ -117,6 +117,28 @@ <h4>{{ subheader }}</h4>
             </div>
             </form>
         </div>
+        {% if 'login' in request.path %}
+            <div class="row mt-3">
+                <div class="col-md-1 col-lg-2"></div>
+                <div class="col-md-10 col-lg-8">
+                    <div class="d-grid gap-1">
+                        {% url "social:begin" "reddit" as social_url %}
+                        {% with social_url|add:'?next=' as social_url %}
+                            <a class="btn btn-outline-danger"
+                                    {% if next %}
+                               href="{{ social_url }}{{ next }}"
+                                    {% else %}
+                               href="{{ social_url }}{{ request.path }}"
+                                    {% endif %}
+                            >
+                                Login with Reddit
+                            </a>
```

Comment:
Why is the login styled as `danger`?

Reply:
because it's red and it's a visual cue closer to Reddit's styling -- if you'd prefer a different color, then we can change it.